Content.Shared/Traits/Assorted/Components/SlipImmunityComponent.cs

```diff
@@ -0,0 +1,11 @@
+using Robust.Shared.GameStates;
+
+namespace Content.Shared.Traits.Assorted.Components;
+
+/// <summary>
+///   This is used for traits that make an entity immune to slips
+/// </summary>
+[RegisterComponent, NetworkedComponent]
+public sealed partial class SlipImmunityComponent : Component
+{
+}
```

Comment:
Redundant trait, you can just give -type: NoSlip to Diona.

Reply:
Resolved [1fe196f](https://github.com/Simple-Station/Einstein-Engines/pull/704/commits/1fe196f024e35a5769f07239f49b926de30070d6)